routers/api/v1/repo/release.go

```diff
@@ -277,7 +277,8 @@ func EditRelease(ctx *context.APIContext) {
 	if form.IsPrerelease != nil {
 		rel.IsPrerelease = *form.IsPrerelease
 	}
-	if err := releaseservice.UpdateReleaseOrCreatReleaseFromTag(ctx.User, ctx.Repo.GitRepo, rel, nil, false); err != nil {
+	// TODO: Support edit/delete attachments
```

Comment:
https://try.gitea.io/api/swagger#/repository/repoDeleteReleaseAttachment

posible

Reply:
I removed this comment since we have that.